manifests/plugin/abrt.pp

```diff
@@ -30,7 +33,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $group                   = $::foreman_proxy::plugin::abrt::params::group,
+  $version                 = $::foreman_proxy::plugin::version,
+  $group                   = $::foreman_proxy::user,
```

Comment:
No, those were intentional and solve it in a much better way by using ```foreman_proxy::settings_file``` instead of doing the same everywhere.